src/core/scene/a-scene.js

```diff
@@ -450,10 +450,20 @@ module.exports.AScene = registerElement('a-scene', {
       value: function () {
         var camera = this.camera;
         var canvas = this.canvas;
-        var embedded = this.getAttribute('embedded') && !this.is('vr-mode');
+        var embedded;
         var size;
         var vrDevice;
         var isVRPresenting;
+        // Workaround for a Webkit bug (https://bugs.webkit.org/show_bug.cgi?id=170595)
+        // where the window does not contain the correct viewport size
+        // after an orientation change. The window size is correct if the operation
+        // is postpoend a few miliseconds.
+        // The if satement below can be removed when the bug is fixed.
+        if (this.iOS) {
```

Comment:
Wouldn't this be a never-ending loop?

Reply:
you looked to an outdated push. Now it is ready for review 😄